### DISCUSSION TITLE
Code name for vE release

### DISCUSSION THREAD

**Discussion by suresh-lc** [lf-edge/edge-home-orchestration-go#592] [Polls]
In few months time the next major release would happen. Hence thought we prepare the code name for the same. Continuing in the alphabetical order, its letter E next in pipeline. Have put down few options for the same. Request your opinion or new suggestions too.

**Comment by MoonkiHong** [comment]
What about Eucalyptus?

**Reply by suresh-lc** [reply]
Had thought it too, to make some unique name, didn't add it . We can sure consider Eucalyptus as also option.
